tools/WixBuild.Tools.targets

```diff
@@ -133,6 +133,11 @@
       Condition=" '$(PleaseSignOutput)'!='' and ('$(SignClientUser)'=='' or '$(SignClientSecret)'=='') "
       Text="Signing is requested but one or both required properites SignClientUser and SignClientSecret were not specified on the command line or as environment variables." />
 
+    <Error
+      Code="WIXBUILD016"
+      Condition=" $(BuildARM64) and !(Exists('$(PlatformSdkArm64LibraryRoot)')) "
+      Text="No suitable Windows SDK found to build ARM64" />
+
```

Comment:
Are we going to force `BuildARM` and/or `BuildARM64` when building in release mode?

Reply:
Well, it'll fail when it tries to build setup...

Reply:
But if you put `BuildARM64` in the right places, it shouldn't fail...

Reply:
Yep, working on it.